docs/docs/development.md

```diff
@@ -129,6 +129,21 @@ Feature flags are enabled on the frontend so that if you're logged in as an admi
 - Change the URL to enable the `future` mode:
     - `http://127.0.0.1:8000/analysis/794236df-2684-4781-bb95-ad9a58293d15/gvl8lg6sayiw/?future=1`
 
+To view the new features without needing to log in you can disable the administrator login requirement by changing:
```

Comment:
Instead of this, how about se add a new setting ALWAYS_SHOW_FUTURE that we can toggle locally, and then we can do this instead:

We can set it to False in settings/base.py, but you can override it settings/local.py to set it to True


```python
"future": settings.ALWAYS_SHOW_FUTURE or (self.request.user.is_staff and bool(self.request.GET.get("future")))
```

Might still be good to document it as you did here, but it makes it much simpler and doesn't require code changes that we'll never commit